rpcs3/Emu/System.cpp

```diff
@@ -279,13 +279,19 @@ std::string Emulator::GetLibDir()
 	return fmt::replace_all(g_cfg.vfs.dev_flash, "$(EmulatorDir)", emu_dir) + "sys/external/";
 }
 
-void Emulator::Load()
+void Emulator::Load(bool add_only)
 {
-	Stop();
+	if (!add_only)
```

Comment:
why stop? we only want to add a few games to list and don't actually interfere with emulation